### ISSUE TITLE
Helloworks should not send initial notification email

### ISSUE THREAD

**Issue by alanna** [opencollective/opencollective#5234]
People still get both emails simultaneously, from OC and HelloWorks. 
![Screen Shot 2022-02-11 at 2 11 33 PM](https://user-images.githubusercontent.com/5498878/153523136-500de7fa-8e1b-4a6c-84e4-c3255c701a8d.png)
![Screen Shot 2022-02-11 at 2 11 20 PM](https://user-images.githubusercontent.com/5498878/153523145-8f26c9dc-075f-4b85-96f7-1a3bb10e3280.png)
 
The normal flow should be:
-     You get an email from Open Collective and click the link to HelloWorks
-     HelloWorks asks you to confirm your email, you click on something like "Send verification"
-     Click on the other email and actually start the flow

 HelloWorks email should only be sent after clicking on the internal email

**Comment by Betree** [comment]
I looked into this quickly. It appears the regression was introduced in https://github.com/opencollective/opencollective-api/pull/6956, it seems that using the delegated authentication is the only way to skip this email. I'll contact HelloWorks support to see what we can do.

**Comment by Betree** [comment]
A bit late, but I sent out this message to HelloWorks. Now waiting for their reply.

> Hi,
> 
> We've been using HelloWorks for more than two years now, and we're pretty happy with it. So first, thanks for this excellent service!
> 
> The account we're using is linked to the email "----------" and runs under the "----------" plan.
> 
> We recently implemented a notification email for the form request because our users were confused by the invitation email from a third-party service they were unaware of. We're now facing an issue where they receive two emails simultaneously: one from us and one from HelloWorks.
> 
> There are two ways we could solve that:
> Ideally, we would like to use the "Delegated authentication" feature (as described in https://www.hellosign.com/products/helloworks/pricing). Our code is ready to handle this, so enabling the feature on our account would 100% solve the problem on our side.
> If not possible, we would like to have a parameter in the `createInstance` API method to prevent the default email from being sent. People would have to reverify their email through HelloWorks, but the communication would still be clearer.
> 
> Thank you in advance,
> Benjamin

**Comment by Betree** [comment]
No reply from HelloWorks to this date. I'll send the message again.

**Comment by alanna** [comment]
@Betree this is still happening. Did we ever hear back?

**Comment by Betree** [comment]
I did, forgot to update this thread cause the reply did not help much:

TLDR ; move to the "Standard plan".

> Hello, 
>
> Thank you for reaching out. Taking a closer look at your team's account, it looks like you current plan level does not have access to delegated authentication. 
> 
> Your current HelloWorks plan is the Essentials 100 plan. The minimum plan level required for delegated authentication is the Standard 1k plan. That said, access to delegated authentication will require an update from Essentials 100 to Standard 1k.
>  
> If you'd like more information on the Standard 1k plan, or if you'd like to move forward with upgrading, please let me know and I'll be happy to connect you with someone that can assist with that process.
>  
> Please feel free to let me know if you have any questions, I will keep an eye out for your reply.
>  
> 
> Best,
> HelloWorks Technical Support 

Now something worth mentioning: in the last week, HelloWorks seems to have gone through a complete rebranding from [HelloSign](https://webcache.googleusercontent.com/search?q=cache:jahpYOksMCQJ:https://fr.hellosign.com/products/helloworks/pricing&cd=1&hl=fr&ct=clnk&gl=fr&client=firefox-b-d) to [Dropbox Sign](https://fr.hellosign.com/products/helloworks).

The "Essentials" plan disappeared, but the standard plan moved from $100/m to $25/m (billed by user). I believe we're currently paying $20/m, so it might be a good time to migrate.

![image](https://user-images.githubusercontent.com/1556356/199922029-aa6eb6d4-24cb-49e5-b65c-26c3df008608.png)

**Comment by alanna** [comment]
@alinamanko what plan are we on now?

**Comment by alinamanko** [comment]
@alanna we are on Essntials 100 - allowing us to send 100 forms per month

**Comment by Betree** [comment]
Since we moved to a higher plan, I'm gonna try and give delegated authentication another shot

**Comment by Betree** [comment]
> Since we moved to a higher plan, I'm gonna try and give delegated authentication another shot

It didn't work:

```
Failed to initialize tax form for account _____ Please contact HelloSign to upgrade to advanced features
```

Quite frustrating since we're now paying an expensive price for the service. I'm going to rollback https://github.com/opencollective/opencollective-api/pull/8483 and contact their support.

**Comment by Betree** [comment]
We're not using HelloWorks anymore.
